test/omopcdmext.jl

```diff
@@ -0,0 +1,17 @@
+@testset "Simple HealthTable OMOP CDM Extension Test" begin
+    person_df = DataFrame(
+        person_id=1,
```

Comment:
Can you also have a test where it checks if it errors? Like a type that is in the field is wrong per the specification?